### PR TITLE
OSSM-6901: Create banner for Service Mesh 3.0 Tech Preview release

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): OCP 4.14+

Issue:
https://issues.redhat.com/browse/OSSM-6901

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by rh-tokeefe** [comment]
@kalexand-rh and/or @aireilly could one of you merge this PR when you have a moment?

PR was rebased on 8/21/24.

**Comment by rh-tokeefe** [comment]
Rebased PR on 8/26/24/

File: _templates/_page_openshift.html.erb [hunk 1]
```diff
@@ -513,6 +528,7 @@
     'openshift-builds' : ['docs_builds', version],
     'openshift-gitops' : ['docs_gitops', version],
     'openshift-lightspeed' : ['docs_lightspeed', version]
```
Comment (aireilly) [inline _templates/_page_openshift.html.erb:530]:
```suggestion
    'openshift-lightspeed' : ['docs_lightspeed', version],
```

Comment (rh-tokeefe) [reply _templates/_page_openshift.html.erb]: Thank you, @aireilly . Just pushed the change.

**Pull request merged**